pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -425,7 +425,7 @@ func (s *Service) SendExchangeResponse(exchangeResponse *Response, destination s
 func (s *Service) marshalAndSend(data interface{}, errorMsg, destination string) (string, error) {
 	// TODO an outboundtransport implementation should be selected based on the destination's URL.
 	// TODO add Destination struct
-	return "", s.outboundDispatcher.Send(data, nil)
+	return "", s.outboundDispatcher.Send(data, "", nil)
```

Comment:
We dont need this function. If your change is going before i think you should just remove it.

Reply:
ok I will remove it

Reply:
If I remove it will effect other functions

Reply:
maybe you remove it in your pr

Reply:
okay i have already removed it in my pr  and made required changes. Ignore it for now